tests/testsuite/bad_config.rs

```diff
@@ -330,6 +331,9 @@ fn bad_dependency_in_lockfile() {
 
 #[cargo_test]
 fn bad_git_dependency() {
+    if uses_gitoxide() {
+        return;
+    }
```

Comment:
To me this reads as "if the test uses gitoxide, return" but its really "if cargo is using gitoxide".  I wonder if "uses_gitoxide"s name should be clarified.

Bonus points for updating `cargo-test-macros` to add a new annotation for marking the test a ignored in this case (though that would cause recompilations when enabling/disabling so unsure if that is fully appropriate).

Reply:
A good point. I thought that `cargo_uses_gitoxide()` might be better. The function was renamed in 7aa1e5c50d55250d32fd48c330c4d4ec861ee911.

> Bonus points for updating cargo-test-macros to add a new annotation for marking the test a ignored in this case (though that would cause recompilations when enabling/disabling so unsure if that is fully appropriate).

It is used to skip tests only once in `avoid_using_git()` and I hope with the new name it's acceptable.

Reply:
Thats fine.  The name doesn't have to be perfect; its just test code.  Just figured a slight tweak would be nice regardless of the direction.